recipes/exceptiongroup/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge exceptiongroup_dev
```

Comment:
Since this is still v1.0.0rc2, I'm sending it to the `exceptiongroup_dev` label.  I will make a `dev` branch from `main` as soon as the feedstock is created to keep this clean.